conda.recipe/meta.yaml

```diff
@@ -8,14 +8,16 @@ source:
 requirements:
   build:
     - python
-    - "taxcalc>=2.1.0"
+    - "taxcalc>=2.4.0"
     - setuptools
+    - paramtools
     - xlrd
 
   run:
     - python
-    - "taxcalc>=2.1.0"
+    - "taxcalc>=2.4.0"
     - setuptools
+    - paramtools
     - xlrd
```

Comment:
@jdebacker you should specify "paramtools>=0.7.1" right here.

Reply:
Got it - thanks.